modules/azure/vnet/variables.tf

```diff
@@ -38,3 +38,51 @@ variable "external_worker_subnet_id" {
   type    = "string"
   default = ""
 }
+
+#variable "etcd_lb_ip" {
```

Comment:
Do we need etcd_lb_ip? Doesn't seem like its being used. If so, lets clean it up and its refs